app/controllers/search_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SearchController < ApplicationController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable @q [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)

<!-- Comment by [SideCI](https://sideci.com) -->